source/CommonJobs/CommonJobs.Application.EvalForm/Commands/GetEvaluationCalifications.cs

```diff
@@ -94,7 +94,8 @@ private CalificationsDto GetEvaluatorEvaluation(CalificationsEvaluationDto evalu
             {
                 View = UserView.Evaluation,
                 Evaluation = evaluationDto,
-                Califications = califications.Where(c => _loggedUser == c.EvaluatorEmployee || c.Owner == CalificationType.Auto).ToList()
+                Califications = califications.Where
+                (c => _loggedUser == c.EvaluatorEmployee || c.Owner == CalificationType.Auto || (evaluationDto.ReadyForDevolution && c.Owner == CalificationType.Company)).ToList()
```

Comment:
@AlphaGit Will fix the styling.

About the logic involved: The evaluator will get one of this califications; his own, the auto evaluation, and the company one ONLY if the evaluation is in the ready for devolution state. 

Will not be receiving other evaluators evaluations since none of the ifs is allowing it.

Will get his own: _loggedUser == c.EvaluatorEmployee

Will get the auto eval: || c.Owner == CalificationType.Auto 

Will get the company evaluation: || (evaluationDto.ReadyForDevolution && c.Owner == CalificationType.Company)